libevmasm/Assembly.cpp

```diff
@@ -599,25 +599,25 @@ LinkerObject const& Assembly::assemble() const
 		switch (i.type())
 		{
 		case Operation:
-			ret.bytecode.push_back((uint8_t)i.instruction());
+			ret.bytecode.push_back(static_cast<uint8_t>(i.instruction()));
 			break;
 		case PushString:
 		{
-			ret.bytecode.push_back((uint8_t)Instruction::PUSH32);
+			ret.bytecode.push_back(static_cast<uint8_t>(Instruction::PUSH32));
 			unsigned ii = 0;
-			for (auto j: m_strings.at((h256)i.data()))
+			for (auto j: m_strings.at(h256(i.data())))
 				if (++ii > 32)
 					break;
 				else
-					ret.bytecode.push_back((uint8_t)j);
+					ret.bytecode.push_back(uint8_t(j));
```

Comment:
Ah, well... that should also probably be a ``static_cast``... but it's not *that* important...